riscv_isa.cpp

```diff
@@ -6,484 +17,794 @@
 #define DEBUG_MODEL
 #include "ac_debug_model.H"
 
-//#  define dbg_printf(args...) fprintf (stderr, "DBG: " args)
-//#define dbg_printf printf();
-
 #define Ra 1
 #define Sp 14
 
+// For using all the RISC-V parameters
 using namespace riscv_parms;
 
 static int processors_started = 0;
-#define DEFAULT_STACK_SIZE (512 * 1024); // check this parameter
+#define DEFAULT_STACK_SIZE (512 * 1024); 
+
 
-// generic instruction behavior method
+// Generic instruction behavior method
```

Comment:
You have changes in no related codes and comments in the same commit.

Move the typos fix to the first commit. If you know the error, fix it in the commit that adds the file.